internal/terminal/table.go

```diff
@@ -11,13 +11,14 @@ import (
 const (
 	logFieldHeaders = "headers"
 	logFieldData    = "data"
+	// gutter is the gap between table columns
```

Comment:
[meganit] add a newline between `logFieldData` and this comment here to break up the const block a little